Eatery Blue/UI/EateryCard/EateryLargeCardContentView.swift

```diff
@@ -19,7 +19,8 @@ class EateryLargeCardContentView: UIView {
     private let labelStackView = UIStackView()
     private let titleLabel = UILabel()
     private let subtitleLabels = [UILabel(), UILabel()]
-    private var favoriteButton = ButtonView(content: UIImageView())
+    private var favoriteButton = ButtonView(content: UIView())
+    private var favoriteButtonImage = UIImageView()
```

Comment:
I don't think the values for these variables are changed anywhere. Can we change these to constants?